app/controllers/insights_cloud/hits_controller.rb

```diff
@@ -22,6 +23,14 @@ def model_of_controller
       ::InsightsHit
     end
 
+    def save_setting_and_sync
```

Comment:
```suggestion
    def save_token_and_sync
```
I think it adds clarity.

Besides that, I think the proper place for this would be the settings controller, and the response could do a redirect to `insights_cloud_hits_url`

Reply:
Where would the sync fit in this way?

Reply:
I think it will still fit to the settings page. Imagine we want other processes start immediately after saving the token.

In later stages, we will refactor the direct call to the sync task into a separate object (maybe a dynflow task) that will be responsible for handling all "state initialization" process.